examples/pytorch/speech-recognition/README.md

```diff
@@ -360,6 +369,10 @@ recognition on one of the [official speech recognition datasets](https://hugging
 ### Whisper Model
 We can load all components of the Whisper model directly from the pretrained checkpoint, including the pretrained model weights, feature extractor and tokenizer. We simply have to specify our fine-tuning dataset and training hyperparameters.
 
+> **Tip:** If you see errors about “dataset scripts are no longer supported”, make sure your
+> `--dataset_name` points to a Hub dataset like `mozilla-foundation/common_voice_XX_X`
+> (e.g., `mozilla-foundation/common_voice_17_0`) rather than a local script.
```

Comment:
```suggestion
> [!TIP]
> If you see errors about “dataset scripts are no longer supported”, make sure your
> `--dataset_name` points to a Hub dataset like `mozilla-foundation/common_voice_XX_X`
> (e.g., `mozilla-foundation/common_voice_17_0`) rather than a local script.
```